polymesh_schema.json

```diff
@@ -281,13 +281,13 @@
             "Community"
         ]
     },
-    "MIP": {
-        "index": "MipsIndex",
+    "PIP": {
+        "index": "PipsIndex",
```

Comment:
This field has been renamed to   `id`, and I've just realized I forget to rename it in our schema